ui/app/rif/pages/domainsDetailPage.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckCircle, faArchive, faBolt, faChevronLeft, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+import { getIconForToken } from '../utils/utils'
+import { CustomButton } from '../components'
+import rifActions from '../../rif/actions'
+
+class DomainsDetailScreen extends Component {
+	constructor(props) {
+		super(props);
+		let resolvers = []
+		Object.assign(resolvers, props.resolvers);
+		this.state = { 
+			resolvers: resolvers,
+			selectedResolverIndex: 0,
+		};
+	}
+	navigateTo (url) {
+		global.platform.openWindow({ url })
+  	}
+
+	render () {
+		const { status, domainName, address, content, expirationDate, autoRenew, ownerAddress, isOwner, isLuminoNode, isRifStorage, domain } = this.props
+		return (
+		<div className={'body'}>
+			<FontAwesomeIcon icon={faChevronLeft} className={'rif-back-button'} onClick={() => this.props.goBack()}/>
+			{status === 'active' && 
```

Comment:
If this page keeps growing, it would be nice to have auxiliar render functions. For example: 

		`	{status === 'active' && this.renderActiveDomain()`

Reply:
Agree

Reply:
Or just create a new ActiveDomain component, that would be better.

Reply:
Ok, let's create a component for each, and have this father-component to render each one with a condition.